File: Tech Module/Software Technologies/Exam_Preparation_02/TeisterMask JavaScript/models/Task.js
const mongoose = require('mongoose');

let taskSchema = mongoose.Schema({
	// TODO: Implement me...

    title: {type: 'string', required: 'true'},
    status: {type: 'string', required: 'true'}
});

let Task = mongoose.model('Task', taskSchema);

module.exports = Task;

File: Tech Module/Software Technologies/Exam_Preparation_01/C# TODO List/TodoList/Models/Task.cs
using System.ComponentModel.DataAnnotations;

namespace TodoList.Models
{
    public class Task
    {
        //TODO: Implement me...

        [Key]
        public int Id { get; set; }

        [Required]
        public string Title { get; set; }

        [Required]
        public string Comments { get; set; }
    }
}